### ISSUE TITLE
Error with avx : ROUNDTRIP_FAILURE

### ISSUE THREAD

**Issue by seblucas** [dropbox/lepton#15]
When trying `lepton-avx` on my Core i7 2760QM which is said to support AVX, I have the following error : 

```
lepton v1.0-
Input Size 6632440 != Roundtrip Size 0
ebde2fe7982b79ad101495a1bdbe4e90 != 0600000005000000ffffffff00000000
ROUNDTRIP_FAILURE
```

Running `lepton.exe` is completely fine.

**Comment by danielrh** [comment]
what compiler did you build it with? What settings?
does make check pass?

**Comment by seblucas** [comment]
I did not compile it, I took it from your [release](https://github.com/dropbox/lepton/releases/tag/1.0)
I use default settings : `lepton-avx img.jpg img.lep`
I did not try to compile it. I'll do it this weekend.

**Comment by danielrh** [comment]
what image did you try it on?

I assume it didn't give you an output file...
it's definitely not designed to work on every image

you could try giving it more memory:
lepton-avx.exe -memory=1024M -threadmemory=256M -allowprogressive

**Comment by danielrh** [comment]
This should be fixed in the latest (1.2) release
Let me know if it's still broken and we'll reopen and reinvestigate

**Comment by seblucas** [comment]
So I tried again with 1.2 and sadly I have the same error
![img_2852](https://cloud.githubusercontent.com/assets/437682/16999491/8d2019b2-4ebe-11e6-803f-57269ba48190.JPG)

Here is the full log : 
```
C:\Partage>lepton-avx.exe IMG_2853.JPG IMG_2852.lep
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
Input Size 5751161 != Roundtrip Size 0
dac674cdedbff8a42c41dcee59138119 != 0600000005000000ffffffff00000000
UNSUPPORTED_JPEG
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
filetype of file "-" is unknown bytes needed to decompress this file
UNSUPPORTED_JPEG
```

If I use lepton.exe, it works fine : 
```
C:\Partage>lepton.exe IMG_2853.JPG IMG_2852.lep
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
14845992 bytes needed to decompress this file
4540528 5751161
78.95%
lepton v1.0-08c52d9280df3d409d9246df7ff166dd94628730
4279814452 bytes needed to decompress this file
4540528 5751161
78.95%
```
